src/main/target/common_pre.h

```diff
@@ -105,6 +105,9 @@
 #define USE_ITCM_RAM
 #define USE_FAST_RAM
 #define USE_DSHOT
+#define USE_DSHOT_TELEMETRY
+#define USE_DSHOT_TELEMETRY_STATS
+#define USE_DYN_IDLE
```

Comment:
`#define USE_RPM_FILTER` is missing, without this `#define USE_DYN_IDLE` is not working

Reply:
Thanks, I'll add the option.

Reply:
thx for fixing this